tests/unittest/kv_cache_manager_v2_tests/test_kv_cache_event_manager.py

```
@@ -0,0 +1,896 @@
+        init_cuda_once,
+        temporary_sys_path,
+    )
+
+try:
+    import torch
+except ImportError:
+    torch = None
+
+
+class _FakePage:
+    def __init__(self, cache_level=CacheLevel(0), priority=0):
```

Ruff (0.15.15)

[warning] Do not perform function call `CacheLevel` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)